packages/kit/src/exports/vite/dev/index.js

```diff
@@ -387,10 +387,14 @@ export async function dev(vite, vite_config, svelte_config) {
 		}
 	});
 
-	// changing the svelte config requires restarting the dev server
-	// the config is only read on start and passed on to vite-plugin-svelte
-	// which needs up-to-date values to operate correctly
 	vite.watcher.on('change', async (file) => {
```

Comment:
could you add a comment here and/or 30 lines above where we have `watch('change', (file) => { ...`? From a quick glance, I'm not sure what the difference is between them and why we need two file watchers

Reply:
Good idea. Found out these all use the same `vite.watcher` but `watch` only runs for certain files